Twitch/Twitch.Base/Services/NewAPI/SubscriptionsService.cs

```diff
@@ -56,5 +56,16 @@ public async Task<SubscriptionModel> GetSubscription(UserModel broadcaster, User
             IEnumerable<SubscriptionModel> subscriptions = await this.GetPagedDataResultAsync<SubscriptionModel>("subscriptions/user?broadcaster_id=" + broadcaster.id + "&user_id=" + user.id);
             return (subscriptions != null) ? subscriptions.FirstOrDefault() : null;
         }
+
+        /// <summary>
+        /// Gets the total number of subscribers for a channel.
+        /// </summary>
+        /// <param name="broadcaster">The channel to get the total number of subscribers for</param>
+        /// <returns>The total number of subscribers for the channel</returns>
+        public async Task<long> GetBroadcasterSubscribersCount(UserModel broadcaster)
```

Comment:
Since all of these are for the broadcaster itself, you don't need to include "Broadcaster" in the name of the method. It can just be "GetSubscribersCount" or "GetSubscriptionsCount".

Reply:
Good point, will update.